agent/src/agent/pipeline/config/stages/source/jdbc.py

```diff
@@ -13,6 +13,8 @@ def _get_config(self) -> dict:
 
     def get_query(self):
         if isinstance(self.pipeline, pipeline.TestPipeline):
+            if not self.pipeline.query:
+                return 'SELECT * FROM t'
```

Comment:
why there might not be a query and what is t?

Reply:
This is for validating the connection when we create a source. We don't have a query then yet, so this is just a dummy query because the streamsets requires to put something in the query field